pkg/dss/scd/models/operations.go

```diff
@@ -35,3 +36,39 @@ type Operation struct {
 	Cells          s2.CellUnion
 	SubscriptionID ID
 }
+
+// AdjustTimeRange adjusts the time range to the max allowed ranges on an
+// Operation.
+func (o *Operation) AdjustTimeRange(now time.Time, old *Operation) error {
+	if o.StartTime == nil {
+		// If StartTime was omitted, default to Now() for new ISAs or re-
```

Comment:
I think we actually don't want to default to now for Operations; we should just reject if the start time isn't specified.

Reply:
Done.